monkey/infection_monkey/telemetry/batchable_telem_mixin.py

```diff
@@ -0,0 +1,22 @@
+from typing import Iterable
+
+from infection_monkey.telemetry.i_batchable_telem import IBatchableTelem
+
+
+class BatchableTelemMixin:
```

Comment:
Why doesn't this implement the `IBatchableTelem`?

Reply:
Because then it would have to implement everything in ITelem. Mixins are not meant to stand on their own. They add a specific functionality to a class that stands on its own.